tests/openssl_sgx_tcrypto_compat/enc/tests_ecalls.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <stdlib.h>
+#include "include_openssl.h"
+#include "openssl/crypto.h"
+#include "openssl_schema.h"
+#include "openssl_sgx_tcrypto_compat_t.h"
+
+extern int common_digest_tests(openssl_api_param* parameter);
+void ecall_set_rdrand_engine()
```

Comment:
We no longer need this initialization code.

Reply:
Done. Thanks